bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/member/GetMembershipsFlow.kt

```diff
@@ -58,3 +61,23 @@ class GetMembershipsFlow(private val forceRefresh : Boolean = false) : FlowLogic
     }
 }
 
+@StartableByRPC
+class GetMembersFlow(private val forceRefresh : Boolean = false) : FlowLogic<List<PartyAndMembershipMetadata>>() {
+
+    companion object {
+        object GOING_TO_CACHE_OR_BNO : ProgressTracker.Step("Going to cache or BNO for membership data")
```

Comment:
Is there a typo somewhere here?

Reply:
Are you referring to the "cache or BNO"? I wanted to express that cache may or may not be used. But at the same time I didn't feel like investing time and code into finding out what's going to be used on that very occasion.

Reply:
Right, might need rephrasing then as "cache" is red as a verb in this context :)